torch/onnx/symbolic.py

```diff
@@ -1690,10 +1690,10 @@ def narrow(g, input, dim, start, length):
 
 
 @parse_args('v', 'i', 'i')
-def _argmax(g, input, dim, keepdim):
+def argmax(g, input, dim, keepdim):
```

Comment:
What would happen if `dim=None`? Does this need special handling?

Reply:
this is another feature we need to support. if dim is None, if it will do the argmax on the flattened tensor. I will add this support in a different PR.